o3po/includes/class-o3po-primary-publication-type.php

```diff
@@ -1444,8 +1444,10 @@ public function get_the_content( $content ) {
             $content = '';
 
             $content .= '<header class="entry-header">';
-            if($settings->get_plugin_option('page_template_for_publication_posts')==='checked')
-                $content .= '<h1 class="entry-title title citation_title"><a href="#">' . esc_html ( get_the_title( $post_id ) ) . '</a></h1>';
+            if($settings->get_plugin_option('page_template_for_publication_posts')==='checked') {
+                $content .= '<h1 class="entry-title title citation_title"><a href="#">' . esc_html ( get_the_title( $post_id ) ) . '</a></h1>'; 
+            }
+
```

Comment:
I don't think adding the {} here increases readability. I would rather not have such changes that only show up in diff but don't change anything in PRs. Thank you!